R/hyperdrive.R

```diff
@@ -692,3 +692,85 @@ get_child_run_hyperparameters <- function(hyperdrive_run) {
 get_child_run_metrics <- function(hyperdrive_run) {
   hyperdrive_run$get_metrics()
 }
+
+#' Create a child run
+#'
+#' @description
+#' Create a child run. This is used to isolate part of a run into a subsection.
+#' @param parent_run The `HyperDriveRun` object.
+#' @param name An optional name for the child run, typically specified for a "part"
+#' @param run_id An optional run ID for the child, otherwise it is auto-generated.
```

Comment:
can't parent_run be any run?

Reply:
These new child run apis are not limited to hyperdriver. It should be part of run.R

Reply:
Thanks for pointing this out. My bad.